_includes/_boinc_1C.htm

```diff
@@ -30,13 +30,7 @@ <h6>Sources of whitelist data</h6>
                 <h6>Project Selection Tips</h6>
                 <ul>
                     <li>
-                        Choose projects with least competition. Use tools such as <a href="https://gridcoinstats.eu/project">Gridcoinstats</a> to help identify such projects.
-                    </li>
-                    <li>
-                        Select projects most compatible with your computer hardware, don't crunch GPU work with a CPU.
```

Comment:
I'm not a fan of getting rid of this line

Reply:
I think for the average user:

- We do not need to get into optimization. There is no reason to expect that they will be able to pick the most optimal projects right off the bat, even if we provided them all the requisite information. If they want to, they can do additional reading on it.
- This just makes setup contain more choices and complication for them. They may not know what CPU is vs GPU is. Projects don't make it easy to find out which use CPU/GPU, even our own CPU/GPU list on the whitelist is frequently outdated. Or maybe they do and our list is right but they are out of WUs for that hardware type.
- This line about choosing projects with less competition doesn't define how "competition" is defined. Is it Percentage gridcoiners? RAC? etc as a new user I would have no idea how to look at that table.

Reply:
I think we should keep these lines until we do a wholesale rewrite to streamline the guides.

Reply:
I have returned this line to the previous state in my repo, I believe the pull request will update itself automatically?